tests/mmptest/src/MMPTest.cs

```diff
@@ -749,5 +749,19 @@ public void BuildWithObjcArcFlag ()
 			});
 
 		}
+
+		// some users add ',' in their assembly names :( 
+		[Test]
+		public void FunnyAssemblyNameShouldNotFail ()
+		{
+			RunMMPTest (tmpDir => {
```

Comment:
Since you are running only one variant, you can just inline the action here though. The testCore thing is for running multiple.